package/samplers/carbo/README.md

```diff
@@ -0,0 +1,113 @@
+---
+author: Optuna Team
+title: Constrained robust Bayesian optimization of expensive noisyblack-box functions with guaranteed regret bounds
+description: Constrained Bayesian optimization with input uncertainties
+tags: [sampler, bo, bayesian-optimization, constrained-optimization]
+optuna_versions: [4.4.0]
+license: MIT License
+---
+
+## Abstract
+
+This package implements a modified Constrained Adversarially Robust Bayesian Optimization (CARBO) sampler based on [the paper `Constrained robust Bayesian optimization of expensive noisyblack-box functions with guaranteed regret bounds`](https://aiche.onlinelibrary.wiley.com/doi/epdf/10.1002/aic.17857).
+This sampler robustly optimizes a function along with inequality constraints that incurs a noise in its input.
+The algorithm details are described in the `Others` section.
+
+## APIs
+
+class CARBOSampler(BaseSampler):
+def __init__(
+self,
+\*,
+seed: int | None = None,
+independent_sampler: BaseSampler | None = None,
+n_startup_trials: int = 10,
+deterministic_objective: bool = False,
+constraints_func: Callable\[\[FrozenTrial\], Sequence\[float\]\] | None = None,
+rho: float = 1e3,
+beta: float = 4.0,
+local_ratio: float = 0.1,
+n_local_search: int = 10,
```

Comment:
```suggestion
n_local_search: int = 16,
```
How about setting  `n_local_seach` to 8 or 16 because setting it not to be a power of 2 causes the following warning?

```sh
[I 2025-07-04 18:30:14,494] Trial 9 finished with value: 4.482056069279998 and parameters: {'x': 4.889303064250705, 'y': 5.466447546932162}.
/Users/yozaki/.asdf/installs/python/3.12.1/lib/python3.12/site-packages/scipy/stats/_qmc.py:958: UserWarning: The balance properties of Sobol' points require n to be a power of 2.
  sample = self._random(n, workers=workers)
```